langs/base.go

```diff
@@ -55,7 +55,7 @@ func init() {
 	registerHelper(&KotlinLangHelper{})
 
 	// for older versions support backwards compatibility
-	fallBackOlderVersions["ruby"] = &RubyLangHelper{Version: "2.5"}
+	fallBackOlderVersions["ruby"] = &RubyLangHelper{Version: "3.1"}
```

Comment:
Which CI build we are talking about here?

Reply:
@ahjones is it ok to remove older version of ruby? Do we need to update any other documentation for the same.